src/components/create-exam/step2.tsx

```
@@ -179,82 +207,85 @@ export const Step2 = ({ onBack, onPublish }: Step2Props) => {
-                        "relative border rounded-2xl bg-base-white min-h-[240px] max-h-[960px] resize-y overflow-y-auto ring-0 focus-within:ring-2 focus-within:ring-offset-2 focus-within:ring-brand-primary-800",
-                        isOverLimit
-                          ? "border-red-500 focus:ring-red-500 focus:border-red-500"
-                          : "border-greyscale-light-200"
+                <div className="flex-1 gap-4 bg-white p-4 rounded-3xl border border-greyscale-light-200">
+                  <FormItem>
+                    <p className="text-lg font-bold text-brand-primary-950 mb-4 flex items-center gap-2">
+                      <PencilIcon className="size-4 shrink-0" />
+                      About Quiz
+                    </p>
+                    <div className="flex flex-col gap-2">
+                      <FormLabel>Any guidelines or final remarks you'd like to include?</FormLabel>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)